scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/package.scala

```diff
@@ -42,14 +42,15 @@ package object semantic {
               case _ => None
             }
           }
-          val (spath, scontents) = minput match {
+          val (splatformpath, scontents) = minput match {
             case mInput.File(path, charset) if charset == Charset.forName("UTF-8") =>
               path.toRelative(sourceroot).toString -> ""
             case mInput.LabeledString(label, contents) =>
               label -> contents
             case other =>
               sys.error(s"bad database: unsupported input $other")
           }
+          val spath = PathIO.toUnix(splatformpath)
```

Comment:
How would `c:\foo.scala` be translated? Can we really say that `c:/foo.scala` is a unix path?